### ISSUE TITLE
Decrypted text shouldn’t select everything on click

### ISSUE THREAD

**Issue by capndesign** [keybase/keybase-issues#4]
After decrypting a message in the web form, clicking on any of the text highlights everything. I was trying to select just a URL from a message to copy, but it ended up highlighting the whole message.

So, I think it'd be best if there was no JS auto-highlighting of everything after you decrypt.

**Comment by paulka** [comment]
I agree it breaks expectations. It might be more consistent with common practice elsewhere to have a button to select and copy the decoded text. 

In my own case, It's particularly annoying as I often need to decrypt and select a specific substring of decrypted text.

**Comment by zQueal** [comment]
> I was trying to select just a URL from a message to copy, but it ended up highlighting the whole message.

>So, I think it'd be best if there was no JS auto-highlighting of everything after you decrypt.

I think this is a relatively small example, no? Keybase has been in beta for a while, but this is the first time we've heard of it as an issue. That alone should indicate a small demographic who find it troublesome. But I see what you're trying to say.

Maybe a JS element could be added to switch the textarea from one which highlights everything, to a normal textbox with plain text inside, so selective items can be copied and paste? I think that'd be a happy medium. It would involve a single extra click for those that want to copy and paste, but normal behavior resumes and those that want the option would have it.
